packages/webview_flutter/tizen/src/webview.cc

```diff
@@ -946,10 +955,9 @@ FlutterDesktopGpuBuffer* WebView::ObtainGpuBuffer(size_t width, size_t height) {
 
 void WebView::DestructBuffer(void* buffer) {
   if (buffer) {
-    std::lock_guard<std::mutex> lock(mutex_);
-    tbm_surface_destroy((tbm_surface_h)buffer);
-    if (rendered_surface_ == buffer) {
-      rendered_surface_ = nullptr;
+    BufferUnit* unit = tbm_pool_->Get((tbm_surface_h)buffer);
+    if (unit) {
+      tbm_pool_->Release(unit);
```

Comment:
At this moment, tbm_surface_ is not destroyed. was that intended?

Reply:
:ok_hand: 
Main reason of this stuff  is the reuse of tbm_suface.